src/NUnitFramework/nunitlite/TextUI.cs

```diff
@@ -482,19 +482,40 @@ private void DisplayNotRunResults(ITestResult result)
 
         private void DisplayTestResult(ITestResult result)
         {
-            string status = result.ResultState.Label;
+            ResultState resultState = result.ResultState;
+            string fullName = result.FullName;
+            string message = result.Message;
+            string stackTrace = result.StackTrace;
+            string reportID = (++_reportIndex).ToString();
+
+            //if (result.AssertionResults.Count > 0)
```

Comment:
You're right and there is! I'm just experimenting with the "compatible" display there.